packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -173,24 +173,35 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
                                             )}
                                             {isEditMode && (
                                                 <>
-                                                    {!belongsToDashboard && (
-                                                        <Menu.Item
-                                                            icon={
-                                                                <MantineIcon
-                                                                    icon={
-                                                                        IconEdit
-                                                                    }
-                                                                />
-                                                            }
-                                                            onClick={() =>
-                                                                setIsEditingTileContent(
-                                                                    true,
-                                                                )
-                                                            }
-                                                        >
-                                                            Edit tile content
-                                                        </Menu.Item>
-                                                    )}
+                                                    <Tooltip
+                                                        disabled={
+                                                            !belongsToDashboard
+                                                        }
+                                                        label="Charts from dashboards aren't editable"
```

Comment:
`Tile content in charts from dashboards is not editable`
Could you replace it with this @jogeshwar01 ?

Reply:
@joaoviana Yes sure. Replaced.

Here is a screenshot-

![image](https://github.com/lightdash/lightdash/assets/85165953/81589c3f-bb98-4bf6-843e-758779c0974c)